### PR TITLE
Clarify code indents

### PR DESCRIPTION
**Pull request by ncguk**
When I started using this library I copied and pasted the code in the README into the CircuitPython REPL but got stuck because the `print` statement didn't appear to be indented in the code block. I've added four spaces to the appropriate lines to make this clearer.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -32,14 +32,14 @@ Then, construct the thermometer class:
...
-    t = adafruit_mcp9808.MCP9808(i2c)
+      t = adafruit_mcp9808.MCP9808(i2c)
 
 Finally, read the temperature property.
 
 .. code-block:: python
 
-    # and print it out
-    print(t.temperature)
+  # and print it out
```
Comment (tannewt) [inline README.rst]: Please indent this too. It should match the code around it.

Comment (ncguk) [reply README.rst]: Hmm. Adding four spaces to the comment seems to cause the code block to ignore the spaces in both lines. Please ignore the amended pull request for now and let me look into how to fix that.

Comment (tannewt) [reply README.rst]: Oh weird! Maybe the blocks should just be merged together? Thanks for figuring this out!

Comment (ncguk) [reply README.rst]: I think that would be the simplest solution. A quick web search shows that this is something people have run into before and there doesn't appear to be an elegant solution. reST gobbles spaces when all lines have the same indent. I'll amend the pull request.

**Pull request merged**